### PR TITLE
Alaska

### PR DESCRIPTION
**Pull request by askpatrickw**
Alaska State Motto: "North to the Future"

### CODE REVIEW

File: states/alaska/elections.toml [hunk 1]
```diff
@@ -3,7 +3,65 @@ date = 2020-08-18
...
+received_by = 2020-10-04
+sources = ["https://www.elections.alaska.gov/Core/electiondatesandhours.php"]
+
+[20201103.absentee.application]
+received_by = 2020-10-24
+sources = ["https://www.elections.alaska.gov/Core/electioncalendar.php"]
+
+[20201103.absentee]
+in_person_start = 2020-10-19
+postmarked_by = 2020-11-03
```
Comment (tannewt) [inline states/alaska/elections.toml:52]: 2020-11-13 received_by date:

> 11/13/2020 | Deadline to receive absetee ballots mailed from **within the U.S.**, U.S.  territories of Puerto Rico, Guam, the Virgin Islands, American Samoa.`

File: states/alaska/elections.toml [hunk 2]
```diff
@@ -3,7 +3,65 @@ date = 2020-08-18
...
+received_by = 2020-07-19
+sources = ["https://www.elections.alaska.gov/Core/electiondatesandhours.php"]
+
+[20200818.absentee.application]
+received_by = 2020-08-08
+sources = ["https://www.elections.alaska.gov/Core/electioncalendar.php"]
+
+[20200818.absentee]
+in_person_start = 2020-08-03
+postmarked_by = 2020-08-18
```
Comment (tannewt) [inline states/alaska/elections.toml:17]: received_by 2020-08-28

**Review by tannewt — CHANGES_REQUESTED** [review]
Found received_by deadline which is 10 days after election. Shouldn't trigger a reminder but still nice to have.

**Comment by askpatrickw** [comment]
Added the absentee received_by dates

**Review by tannewt — APPROVED** [review]
Thank you!

**Pull request merged**